net/vncrepeater/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2006-2014 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=vncrepeater
+PKG_VERSION:=0.14
+PKG_RELEASE:=1
+PKG_MAINTAINER:=JimmyS83 
+PKG_LICENSE:=GNU GPL
+
+
+PKG_SOURCE:=repeater014.zip
+PKG_SOURCE_URL:=http://jtko.mbnet.fi/uvncrepeater
+PKG_MD5SUM:=3005ebbb2f9442cbea4cbcaa71925dbf
+
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/uclibc++.mk
+include $(INCLUDE_DIR)/package.mk
+
+define Package/vncrepeater
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=$(CXX_DEPENDS)
+  TITLE:=UltraVNC repeater for Linux
+  URL:=http://www.uvnc.com/addons/repeater.html
+endef
+
+define Package/vncrepeater/conffiles
+/etc/vncrepeater.conf
+endef
+
+define Package/vncrepeater/description
+ Viewer can be behind Nat router or directly connected to the internet 
+ instead of forwarding serveral ports, you only need to forward 1 port. 
+ If the PC that runs the Repeater has access to the local DNS server, 
+ you can use your local DNS names instead of 10.10.10.12.
+ This could be handy when you have a dynamic DHCP server allocating 
+ ip adresses for your PC.
+endef
+
+define Build/Prepare
+	mkdir -p $(PKG_BUILD_DIR)
+	$(PKG_UNPACK)
+	$(CP) $(PKG_BUILD_DIR)/../Ver014/* $(PKG_BUILD_DIR)/
+	rm -rf $(PKG_BUILD_DIR)/../Ver014/
+	$(Build/Patch)
+endef
+
+define Build/Compile
+	$(call Build/Compile/Default, \
+		CC="$(TARGET_CXX)" \
+		CFLAGS="$(TARGET_CFLAGS) $(TARGET_CPPFLAGS) -fno-rtti" \
+		repeater \
+	)
+endef
+
+define Package/vncrepeater/install	
+	$(INSTALL_DIR) $(1)/usr/sbin $(1)/etc
```

Comment:
* Directory  ``` $(1)/etc ``` is created twice. On this line and then also two lines later. It is probably more clear if you remove it from this first line.